src/Pages/Profile/admin/AdminView.jsx

```diff
@@ -0,0 +1,412 @@
+import React, { Component } from "react";
```

Comment:
This file is much to complicated. Try breaking it into components, or consolidate the if statements